src/components/content/order/common/ServiceTitle.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { Tooltip, Typography } from 'antd';
+import React from 'react';
+import serviceOrderStyles from '../../../../styles/service-order.module.css';
+
+export const ServiceTitle = ({
+    title,
+    version,
+    icon,
+}: {
+    title: string;
+    version?: string;
+    icon: string;
+}): React.JSX.Element => {
+    const { Paragraph } = Typography;
+    return (
+        <>
+            <Tooltip placement='topLeft' title={title}>
+                <Paragraph ellipsis={true} className={serviceOrderStyles.serviceTitleClass}>
+                    {icon ? <img src={icon} className={serviceOrderStyles.serviceTitleIconClass} alt='icon' /> : ''}
```

Comment:
icon and version are mandatory fields in the interface. Do we really need the conditional handling for both?